pkg/jobs/remove_link.go

```diff
@@ -0,0 +1,110 @@
+package jobs
+
+import (
+	"context"
+
+	"github.com/getsentry/sentry-go"
+	"github.com/go-pg/pg/v10"
+	"github.com/gocraft/work"
+	"github.com/monetr/rest-api/pkg/crumbs"
+	"github.com/monetr/rest-api/pkg/repository"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	RemoveLink = "RemoveLink"
+)
+
+type RemoveLinkJob struct {
+	accountId uint64
+	linkId    uint64
+	userId    uint64
+	log       *logrus.Entry
+	db        *pg.DB
+}
+
+func (r *RemoveLinkJob) Run(ctx context.Context) error {
```

Comment:
Make the `Run(ctx context.Context) error` part of an interface. This should make some basic things a bit easier or atleast force things to be more consistent.